test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -3421,6 +3421,18 @@ def forward(self, x):
         x = torch.randn(3, 3)
         self.run_test(MyReluModule(), x)
 
+    @skipIfUnsupportedMinOpsetVersion(9)
+    def test_clip9(self):
+        class MyClip9(torch.nn.Module):
```

Comment:
```suggestion
        class MyClip6(torch.nn.Module):
```

Reply:
although you can leave the clip-6 inside opset9 file, name it as the correct name to let documented the intended onnx spec